packages/ui/src/components/Designer/Sidebar/DetailView/TextPropEditor.tsx

```diff
@@ -97,13 +100,14 @@ const SelectSet = (props: {
   label: string;
   value: string;
   options: string[];
+  width?: string;
```

Comment:
All `NumberInputSet` have `width` prop.
we don't need to be optional.

Reply:
This is `SelectSet`, not all of them have a width set (Alignment and FontName do not).
We could change this though if you prefer?

Reply:
Oh... sorry.
so you don't need to change.